src/ui/pages/Notifications/components/EarlierNotification.tsx

```diff
@@ -0,0 +1,102 @@
+import { IonButton, IonList } from "@ionic/react";
+import {
+  forwardRef,
+  useCallback,
+  useImperativeHandle,
+  useMemo,
+  useState,
+} from "react";
+import InfiniteScroll from "react-infinite-scroll-component";
```

Comment:
Added another comment regarding the use of this dependency, please use what's recommended on Figma.